src/index.scss

```diff
@@ -57,7 +57,6 @@
 // Vendor Styles
 @forward '../node_modules/font-awesome/css/font-awesome.min.css';
 @forward '@okta/okta-signin-widget/css/okta-sign-in.min.css'; // use path /css instead of /dist/css because that's what the package exports (even though it doesn't match the filesystem)
-@forward '@trussworks/react-uswds/lib/index.css';
```

Comment:
Removing this as per my discussion with @patrickseguraoddball, because its inclusion is overwriting all the overwrites we implemented in `_settings.scss` file. 
Some things in particular were like max-width to the grid containers were smaller than what we currently have, a blue header to all tables, etc.